cmd/worker/wire_gen.go

```diff
@@ -1,8 +1,7 @@
 // Code generated by Wire. DO NOT EDIT.
 
 //go:generate go run github.com/google/wire/cmd/wire
-//go:build !wireinject
-// +build !wireinject
```

Comment:
I know these guys are not related to the PR itself, but they always show up. I'm not saying we need to take these guys out, but do we understand what they do?

Reply:
No clue, going to remove it